README.adoc

```diff
@@ -188,9 +188,11 @@ include::finish/src/main/liberty/config/server.xml[]
 ----
 
 The [hotspot=library]`<library />` tag includes the library reference that indicates
-to the server where the Hazelcast implementation of JCache is located.
+to the server where the Hazelcast implementation of JCache is located. 
+Your Hazelcast implementation of JCache is a JAR file which resides in the location defined by
+the [hotspot=hazelcastLib]`${hazelcast.lib}` variable.
```

Comment:
Change
"Your Hazelcast implementation of JCache is a JAR file which resides in the location defined by the [hotspot=hazelcastLib]`${hazelcast.lib}` variable."
To
"Your Hazelcast implementation of JCache is a JAR file that resides in the location that is defined by the [hotspot=hazelcastLib]`${hazelcast.lib}` variable."